src/components/Sort.jsx

```diff
@@ -0,0 +1,31 @@
+
+
+const Sort = ({setSort, setOrder}) => {
+
+   const submitForm = (event) => {
+         event.preventDefault()
+         
+      
+       setSort(event.target[1].value)
+       setOrder(event.target[2].value) 
+   }
+   
+    return(
+       <form className="sort"  onSubmit={submitForm}>
+        <fieldset>
+            <select key='sort-by'>
+                <option value="created_at">Date</option>
+                <option value="votes">Votes</option>
+                <option value="comment_count">Comment Count</option>
```

Comment:
Also it displays ascending by default, which suggests that the current list is ordered this way. Actually, its order by descending. I'd switch the default value so it reflects how the list is ordered at the start